src/py42/settings/debug.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import sys
+
+
+class _DebugSettings(object):
```

Comment:
this class is exactly the same as it was in `__init__`, moving it here and assigning it to `sys.modules` (see the last line of this file) makes the class be able to be imported via `import py42.settings.debug`. `from py42.settings import debug` also still works.

Reply:
Nice, didn’t know about this possibility.